README.md

```diff
@@ -108,6 +108,12 @@ permissions for these operations. For more information on required permissions,
 [documentation for SAM permissions](
 https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-permissions.html).
 
+### Jinja SAM Templates
```

Comment:
would be nice to provide an example using this with `sceptre_user_data`.  Or maybe you can just change the current example slightly?  something like this..

```
template:
    type: sam
    path: path/from/my/cwd/template.j2
    artifact_bucket_name: {{ template_bucket_name }}
    artifact_prefix: {{ template_key_prefix }}
    build_args:
        use-container: True

sceptre_user_data:
  OpenPorts: ["443", "80"]  
```

Reply:
Yeah, I'll include A jinja example. Good call.

Reply:
Included.